data/data/gcp/bootstrap/main.tf

```diff
@@ -20,6 +20,10 @@ data "ignition_config" "redirect" {
   }
 }
 
+resource "google_compute_address" "bootstrap" {
+  name = "${var.cluster_id}-bootstrap-external-ip"
```

Comment:
https://github.com/openshift/installer/blob/e5b4614cbdbf4a960ddb0d9c3d95792a214793ae/data/data/gcp/network/external-lb.tf#L2

^^ consistency of naming would be great :)

Reply:
i changed "external" to "public" to match this.